cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -436,3 +445,41 @@ pub(crate) fn compute_restore_footprint_transaction_data_and_min_fee(
         bucket_list_size,
     )
 }
+
+fn compute_restore_footprint_rent_changes(
+    footprint: &LedgerFootprint,
+    ledger_storage: &LedgerStorage,
+    min_persistent_entry_expiration: u32,
+    current_ledger_seq: u32,
+) -> Result<Vec<LedgerEntryRentChange>> {
+    let mut rent_changes: Vec<LedgerEntryRentChange> = Vec::new();
+    for key in footprint.read_write.as_vec() {
+        let unmodified_entry = ledger_storage
+            .get(key, true)
+            .with_context(|| format!("cannot get ledger entry with key {key:?}"))?;
```

Comment:
could you rephrase that in a more "actionable" tone ? 
i.e. `The ledger key specified in the footprint read-write section {key:?} cannot be found on chain.`

Reply:
I find that unnecessarily verbose

Reply:
I slightly rephrased it.